packages/odyssey-react-mui/src/themes/odyssey/components.ts

```diff
@@ -0,0 +1,321 @@
+/*!
+ * Copyright (c) 2022-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import type { ThemeOptions } from "@mui/material";
+
+export const components: ThemeOptions["components"] = {
+  MuiCssBaseline: {
+    styleOverrides: {
+      boxSizing: "border-box",
+    },
+  },
+  MuiCircularProgress: {
+    defaultProps: {
+      size: "1.14285714rem",
+      thickness: 8,
+      color: "primary",
+      disableShrink: false,
+      variant: "indeterminate",
+    },
```

Comment:
**NON-BLOCKING**

What's this `1.14rem` value? Are we not calculating it using px->rem? I, as a user, can't tell what `1.14rem` is ever going to be or the intention of it without a base-font-size and pixel representation.

This is personal preference though. I _can_ say that 8 decimal places is a bit much for me to understand if I ever have to change this.